RHEL/7/input/xccdf/system/software/integrity.xml

```diff
@@ -377,4 +377,87 @@ ossrg="SRG-OS-000480-GPOS-00227" stigid="030820" />
+utilize authentication that meets industry and government requirements. For government systems, this allows
+Security Levels 1, 2, 3, or 4 for use on Red Hat Enterprise Linux.
+<br /><br />
+See <b>http://csrc.nist.gov/publications/PubsFIPS.html</b> for more information.
+</description>
+
+<Rule id="package_dracut-fips_installed" severity="medium">
+<title>Install the dracut-fips Package</title>
+<description>
+To enable FIPS, the system requires that the <tt>dracut-fips</tt>
+package be installed. To install the <tt>dracut-fips</tt> package:
+<pre>$ sudo yum install dracut-fips</pre>
+</description>
+<ocil clause="the package is not installed">
+<package-check-macro package="dracut-fips" />
+</ocil>
+<rationale>
+Use of weak or untested encryption algorithms undermines the purposes of utilizing encryption to
+protect data. The operating system must implement cryptographic modules adhering to the higher
+standards approved by the federal government since this provides assurance they have been tested
+and validated.
+</rationale>
+<ident cce="RHEL7-CCE-TBD" />
+<oval id="package_dracut-fips_installed" />
+<ref nist="AC-17(2)" disa="68,2450" stigid="TBD"
+ossrg="SRG-OS-000033-GPOS-00014,SRG-OS-000396-GPOS-00176,SRG-OS-000478-GPOS-00223" />
+</Rule>
+
+<Rule id="grub2_enable_fips_mode" severity="medium">
+<title>Enable FIPS Mode in GRUB2</title>
+<description>
+To ensure all FIPS mode is enabled, rebuild <tt>initramfs</tt> by run the following command:
+<pre>dracut -f</pre>
+After the <tt>dracut</tt> command has been run, add the argument <tt>fips=1</tt> to the default
+GRUB 2 command line for the Linux operating system in
+<tt>/etc/default/grub</tt>, in the manner below:
+<pre>GRUB_CMDLINE_LINUX="crashkernel=auto rd.lvm.lv=VolGroup/LogVol06 rd.lvm.lv=VolGroup/lv_swap rhgb quiet rd.shell=0 fips=1"</pre>
+Finally, rebuild the <tt>grub.cfg</tt> file by using the
+<pre>grub2-mkconfig -o</pre> command as follows:
+<ul>
+<li>On BIOS-based machines, issue the following command as <tt>root</tt>:
+<pre>~]# grub2-mkconfig -o /boot/grub2/grub.cfg</pre></li>
+<li>On UEFI-based machines, issue the following command as <tt>root</tt>:
+<pre>~]# grub2-mkconfig -o /boot/efi/EFI/redhat/grub.cfg</pre></li>
+</ul>
+</description>
+<ocil clause="FIPS is not configured or enabled in grub">
+To verify that FIPS is enabled properly in grub, run the following command:
+<pre>$ grep fips /etc/default/grub</pre>
+The output should contain fips=1
+</ocil>
+<rationale>
+Use of weak or untested encryption algorithms undermines the purposes of utilizing encryption to
+protect data. The operating system must implement cryptographic modules adhering to the higher
+standards approved by the federal government since this provides assurance they have been tested
+and validated.
+</rationale>
+<warning category="general">Running <pre>dracut -f</pre> will overwrite the existing initramfs file.</warning>
```

Comment:
I would add yet another warning here, that after the recommended change is made, it is necessary to reboot the system for the changes to be applied. (Item 5. from https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Security_Guide/sect-Security_Guide-Federal_Standards_And_Regulations-Federal_Information_Processing_Standard.html )